hub/client/client.py

```diff
@@ -157,7 +157,7 @@ def get_dataset_credentials(
         Returns:
             tuple: containing full url to dataset, credentials, mode and expiration time respectively.
         """
-        relative_url = GET_DATASET_CREDENTIALS_SUFFIX % (org_id, ds_name)
+        relative_url = GET_DATASET_CREDENTIALS_SUFFIX.format(org_id, ds_name)
```

Comment:
I couldn't convert this to a fstring as it's a variable, so just decided to go with `.format` instead. Hope that's fine!